packages/client/hmi-client/src/api/api.ts

```diff
@@ -108,9 +108,9 @@ export interface PollerResult<T> {
 }
 
 export class Poller<T> {
-	pollingInterval = 2000;
+	pollingInterval = 3000;
 
-	pollingThreshold = 10;
+	pollingThreshold = 350;
```

Comment:
These are not really sensible defaults (17+ minutes) for a poller. For me it is more natural to actively acknowledge that there is a long running process (calibrate, optimize, funman, ensemble-calibrate), then it is to active acknowledge that there is a short-running process (simulate).

Also note right now simulate will poll for 17+ minutes before it fails over, we should terminate simulate if it exceeds say ... 2 minutes.